### DISCUSSION TITLE
Sonoff TH Elite THR320D | GPIO25 ADC Temp Input  Issue

### DISCUSSION THREAD

**Discussion by pegged-pauses0** [arendst/Tasmota#18683] [Q&A]
Has anyone used the Sonoff TH Elite THR320D as an ADC Temp input? For some reason, my GPIO25 is outputting 3.3V constantly. I have GIO27 enabled for the 3.3V supply, but even with it set to "Output Lo", GPIO25 is still reflecting 3.3V on my multimeter. I have no sensors connected at this stage.

Pin 1 - 3.3V GPIO27
Pin 2 - Data GPIO25 (Always a 3.3V reading on my multimeter)
Pin 3 - NC
Pin 4 - Ground
** Pinout image received from Blakadder site. https://templates.blakadder.com/sonoff_THR320D.html

Using Tasmota v12.5.0.2 Peyton release

**Comment by sfromis** [comment]
Gpio 25 cannot be used for ADC, only gpio 32-39.

**Comment by pegged-pauses0** [comment]
Thank you for the information.
